pom.xml

```diff
@@ -461,274 +461,5 @@
       </plugin>
     </plugins>
   </build>
-  <distributionManagement>
-    <repository>
-      <id>sonatype.release</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
-    </repository>
-    <snapshotRepository>
-      <id>sonatype.snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-    <site>
-      <id>cask</id>
-      <url>http://cask.co</url>
-    </site>
-  </distributionManagement>
-
-  <profiles>
-    <!-- Profile for dist -->
-    <profile>
-      <id>dist</id>
-      <properties>
-        <!--
-          Service name before removing cdap- prefix.
-          It's needed so that children pom can override it.
-          The value is then get transformed into service.name by regex using the build-helper plugin
-        -->
-        <original.service.name>${project.artifactId}</original.service.name>
-        <stage.dir>${project.build.directory}/stage-packaging</stage.dir>
-        <stage.opt.dir>${stage.dir}/opt/cdap/${package.name}</stage.opt.dir>
-        <stage.lib.dir>${stage.opt.dir}/lib</stage.lib.dir>
-        <stage.plugins.dir>${stage.opt.dir}/plugins</stage.plugins.dir>
-        <stage.coprocessor.dir>${stage.opt.dir}/coprocessor</stage.coprocessor.dir>
-        <package.deb.depends>--depends cdap</package.deb.depends>
-        <package.rpm.depends>--depends cdap</package.rpm.depends>
-        <package.deb.arch>all</package.deb.arch>
-        <package.rpm.arch>all</package.rpm.arch>
-        <package.dirs>opt etc</package.dirs>
-        <package.config.dirs>--config-files etc/cdap --config-files etc/logrotate.d --config-files etc/security</package.config.dirs>
-      </properties>
-      <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-jar-plugin</artifactId>
-              <version>2.4</version>
-              <executions>
-                <execution>
-                  <id>jar</id>
-                  <phase>prepare-package</phase>
-                  <configuration>
-                    <outputDirectory>${stage.lib.dir}</outputDirectory>
-                    <finalName>${project.groupId}.${project.build.finalName}</finalName>
-                  </configuration>
-                  <goals>
-                    <goal>jar</goal>
-                  </goals>
-                </execution>
-              </executions>
-            </plugin>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
This second plugins section is removed because it's invalid to have two plugins sections. This one contains a bunch of stuff that looks like it was copied directly from the CDAP project that is not used here.